### PR TITLE
Feature/update post

### PR DESCRIPTION
**Pull request by teamchz**

### CODE REVIEW

**Review by thunyathep — APPROVED** [review]

**Review by sura1542 — APPROVED** [review]

File: lib/src/data/respository/post_repository.dart [hunk 1]
```diff
@@ -19,7 +13,7 @@ class PostRepository {
     print(token);
```
Comment (aofattaporn) [inline lib/src/data/respository/post_repository.dart]: I think maybe null value, pls check it

File: lib/src/screens/Utilities/ToastSuccess.dart [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//   @override
+//   Widget build(BuildContext context) {
+//
+//     return Fluttertoast.showToast(
+//         msg: 'สถานะการโพสสำเร็จ',
+//         fontSize: 18,
+//         gravity: ToastGravity.TOP
+//     );;
+//   }
+// }
```
Comment (aofattaporn) [inline lib/src/screens/Utilities/ToastSuccess.dart]: pls remove it

**Review by aofattaporn — COMMENTED** [review]
pls check before pushing again

**Review by aofattaporn — APPROVED** [review]
LGTM

**Review by Balloii — COMMENTED** [review]
LGTM

**Pull request merged**